tests/unit/test_building_utils.py

```diff
@@ -131,34 +125,13 @@ def test_add_suffix_to_extension():
 
 def test_should_include_system_binary():
     CASES = [
-        ('lib-dynload/any',
-         '/usr/lib64/any',
-         [],
-         True),
-        ('libany',
-         '/lib64/libpython.so',
-         [],
-         True),
-        ('any',
-         '/lib/python/site-packages/any',
-         [],
-         True),
-        ('libany',
-         '/etc/libany',
-         [],
-         True),
-        ('libany',
-         '/usr/lib/libany',
-         ['*any*'],
-         True),
-        ('libany2',
-         '/lib/libany2',
-         ['libnone*', 'libany*'],
-         True),
-        ('libnomatch',
-         '/lib/libnomatch',
-         ['libnone*', 'libany*'],
-         False),
+        ('lib-dynload/any', '/usr/lib64/any', [], True),
```

Comment:
These cases were also intentionally structures, although here the loss of structure is not as pronounced.